lib/cc/engine/analyzers/violation.rb

```diff
@@ -57,7 +61,15 @@ def name
         end
 
         def calculate_points
-          base_points * issue.mass
+          if issue.mass > threshold
```

Comment:
Thanks, I didn't realize that. Making change.

Anyway, interestingly, since the `DEFAULT_POINTS` = `1_500_000`, it's the same result on either leg of conditional. 

But your suggestion is much clearer :) thanks